daemon/logger/jsonfilelog/read.go

```diff
@@ -2,6 +2,7 @@ package jsonfilelog
 
 import (
 	"bytes"
+	"context"
```

Comment:
this is not compatible with `go1.6` I think, you should use `golang.org/x/net/context` I think.